js/google_ima.js

```diff
@@ -1191,6 +1229,12 @@ require("../html5-common/js/utils/utils.js");
               }, this), 100);
             }
             break;
+          case eventType.PAUSED:
+            if (this.videoControllerWrapper)
```

Comment:
From the main_html5 plugin side I can say that the events are not raised while paused.
I think the only harm they will have is possibly raising events that aren't needed, which may be a big issue or a small one depending on performance impact.  I'm guessing it will be fine either way